### PR TITLE
Fixes Issue #46 

### PR DESCRIPTION
**Pull request by YBirhane**

### CODE REVIEW

File: ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst [hunk 1]
```diff
@@ -3,19 +3,89 @@
     International License. To view a copy of this license, visit
     http://creativecommons.org/licenses/by-sa/4.0/.
 
-Creating a Scatter Plot in Sheets {#creating-a-scatter-plot-in-sheets}
+Creating a Scatter Plot in Sheets 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst]: To be consistent with other files, use the following notation for headings

```
Creating a Scatter Plot in Sheets
=================================
```

File: ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst [hunk 2]
```diff
@@ -3,19 +3,89 @@
...
+In this section, you will learn how to make a scatter plot in Sheets by
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst]: Remove "by following along with a tutorial"

File: ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst [hunk 3]
```diff
@@ -3,19 +3,89 @@
...
-variable. Make sure to customize your chart by adding vertical and
-horizontal axis labels. This scatterplot is similar to the first, in
-that as latitude increases, the mean temperature drops. In this chart,
-however, the drop in temperature is not as dramatic as it was for the
-mean January temperatures. In the next section, you’ll learn how to
-describe this difference with more precision.
+
+How to make a scatter plot
+--------------------------
+First, copy over the city, state, latitude and July mean temperature columns
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst:18]: Blank line after rows of -s

File: ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst [hunk 4]
```diff
@@ -3,19 +3,89 @@
...
-however, the drop in temperature is not as dramatic as it was for the
-mean January temperatures. In the next section, you’ll learn how to
-describe this difference with more precision.
+
+How to make a scatter plot
+--------------------------
+First, copy over the city, state, latitude and July mean temperature columns 
+from the weather data here into a new sheet.
+
+.. image:: figures/create_a_scatter_copy_data.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst:21]: Missing :align: center for all images

File: ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst [hunk 5]
```diff
@@ -3,19 +3,89 @@
...
+.. image:: figures/create_a_scatter_axistitle1.png  
+   :alt: A screenshot of how to change the axis titles. 
+
+
+Then select “Horizontal axis title” and “Vertical axis title” respectively to 
+add a descriptive horizontal axis label and a descriptive vertical axis label.
+
+.. image:: figures/create_a_scatter_horizontal_label.png
+   :alt: A screenshot of how to change the horizontal axis title. 
+.. image:: figures/create_a_scatter_vertical_label.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/creating_a_scatter_plot_in_sheets.rst:96]: Blank lines between images

**Review by raskutti — CHANGES_REQUESTED** [review]
Overall LGTM. A few minor comments.

**Comment by YBirhane** [comment]
@raskutti  everything should be fixed now!

**Review by raskutti — APPROVED** [review]

**Pull request merged**